backend_new/src/modules/application.module.ts

```diff
@@ -3,11 +3,24 @@ import { ApplicationController } from '../controllers/application.controller';
 import { ApplicationService } from '../services/application.service';
 import { PrismaModule } from './prisma.module';
 import { EmailModule } from './email.module';
+import { ListingModule } from './listing.module';
+import { MultiselectQuestionModule } from './multiselect-question.module';
+import { ApplicationCsvExporterService } from '../services/application-csv-export.service';
+import { UnitTypeService } from '../services/unit-type.service';
 
 @Module({
-  imports: [PrismaModule, EmailModule],
+  imports: [
+    PrismaModule,
+    EmailModule,
+    ListingModule,
+    MultiselectQuestionModule,
+  ],
   controllers: [ApplicationController],
-  providers: [ApplicationService],
+  providers: [
+    ApplicationService,
+    ApplicationCsvExporterService,
+    UnitTypeService,
```

Comment:
we should import the module instead of the service. Each time a module brings in the provider a new instance of the service is spun up, but the module just injects its already spun up instance of the provider

Reply:
removed the need to specify UnitTypeService here